self-tests.yml

```diff
@@ -650,4 +653,4 @@ buildvariants:
       goos: linux
       goarch: amd64
     tasks:
-      -  name: "docker-cleanup"
\ No newline at end of file
+      -  name: "docker-cleanup"
```

Comment:
What changed?

Reply:
This is a whitespace change done by my editor because in POSIX, [a line has to end in a newline or else it's not actually a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206).

Reply:
cool